react-native-scripts/template/README.md

```diff
@@ -120,3 +121,15 @@ If you have made use of Expo APIs while working on your project, then those API
 ## Tips and Tricks
 
 * If you have a local network the prevents your phone from accessing the address that is printed when running the packager, you can often work around this issue by "tethering" your computer to your phone's internet connection. Some phones call this feature "mobile hotspot." While it's very convenient, if you have a data plan where you pay for data transfer you should be careful when using this feature, especially when installing dependencies from NPM.
+
+## Troubleshooting
+
+* **QR code doesn't work when using Webstorm's built-in terminal**
```

Comment:
This should be a three-hash heading, like so:

```
### QR code won't scan on some terminal themes
```

I think it would be good to phrase this as "some terminal color schemes, including WebStorm's default dark one" rather than making it WebStorm specific. Someone could easily install their own terminal theme which makes QRs not scan. For example if your terminal theme set both light gray and dark gray to the same color, then it probably would just show up as a single square of color.